www/public-interface.js

```diff
@@ -95,6 +100,24 @@ module.exports = function init(exec, cookieHandler, urlUtil, helpers, globalConf
 
   function setRequestTimeout(timeout) {
     globalConfigs.timeout = helpers.checkTimeoutValue(timeout);
+    globalConfigs.connectTimeout = helpers.checkTimeoutValue(timeout);
+    globalConfigs.readTimeout = helpers.checkTimeoutValue(timeout);
```

Comment:
Hmm okay, but you don't seem to use `globalConfigs.timeout` anywhere, right? And changing `readTimeout` will change the request timeout on iOS. Did I get this right?